stable/fluent-bit/values.yaml

```diff
@@ -5,8 +5,8 @@ on_minikube: false
 image:
   fluent_bit:
     repository: fluent/fluent-bit
-    tag: 1.0.6
-  pullPolicy: IfNotPresent
+    tag: 1.1.0
+  pullPolicy: Always
```

Comment:
Is the always policy needed by default ?

Reply:
Ideally yes (maintainers stats), this was changed by mistake in a previous version, user can always adjust it as required .

Reply:
Ok, thanks for the answer!